### PR TITLE
Noarch package

### PR DESCRIPTION
**Pull request by orbeckst**
Change the MDAnalysisTests package to **noarch** (see https://github.com/MDAnalysis/mdanalysis/issues/3727)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by orbeckst** [comment]
@conda-forge-admin, please rerender

**Comment by IAlibay** [comment]
@orbeckst if you want to re-render you'll have to either re-render locally or open a new PR from a personal fork unfortunately

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```
Comment (IAlibay) [inline recipe/meta.yaml:15]: I believe the way forward here is to have noarch in the build section: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (orbeckst) [reply recipe/meta.yaml:15]: Ok, I went with what we have for griddataformats https://github.com/conda-forge/griddataformats-feedstock/blob/main/recipe/meta.yaml

**Comment by orbeckst** [comment]
WIll do this from personal fork. Sigh. Thanks @IAlibay

**Pull request closed without merging**